superbit_lensing/ngmix_fit/ngmix_fit.py

```diff
@@ -11,25 +11,25 @@
 
```

Comment:
Is `NGMixMEDS' substantially different from `meds.MEDS`?

Reply:
actually I'm not sure - I did this to follow what `ngmix` does in its most recent examples, but downgraded the API calls to v1.3.9

I wrote this 6+ months ago so I don't really remember though. It is currently only used for bdf fits run as an afterburner

Reply:
OK, cool. And just to confirm for posterity: at present, the `bdf` fits are just for photometry right now, right?

Reply:
yep!